frontend/src/app/profile/profile-routing.module.ts

```diff
@@ -1,18 +1,15 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { ProfileLandingPage } from "./profile-landing.page";
```

Comment:
Instead of landing page profile page is shown. Landing page was there because I was using child routes and a ion router outlet, which was unnecessary I suppose.